src/pages/Login/style.css

```diff
@@ -1,3 +1,12 @@
 .App-login {
     width: 400px;
+    background-color: rgba(69, 156, 209, 0.5);
```

Comment:
non footer related changes would be better in another branch

Reply:
This was also pushed into the feature/login-page branch, which hasn't been merged yet